build-tools/dependencies/dependencies.projitems

```diff
@@ -5,15 +5,15 @@
     <_AptGetInstall>apt-get -f -u install</_AptGetInstall>
   </PropertyGroup>
   <ItemGroup>
-    <RequiredProgram Include="$(HostCcName)" />
-    <RequiredProgram Include="$(HostCxxName)" />
+    <RequiredProgram Include="$(HostCcName)" Condition=" '$(HostOS)' != 'Windows' " />
```

Comment:
Let's talk about "beauty": I think code should be pretty.

"Pretty" is not well defined, but to me, it's this: things should be horizontally aligned whenever practical.

Specifically, these `Condition` attributes: note that [*every other* `Condition` attribute in this file is horizontally aligned](https://github.com/xamarin/xamarin-android/blob/c0f31f8/build-tools/dependencies/dependencies.projitems).

Would you mind fixing the indentation here? :-)